tool/src/js2/formatter.rs

```diff
@@ -0,0 +1,238 @@
+use std::borrow::Cow;
```

Comment:
nit: make sure to run `cargo fmt` on this code

Reply:
Done!